commit 46d439b17f22216ce8f9257a982c6ade5d1c5931
Author: Vince Busam <vbusam@google.com>
Date:   Tue Feb 26 13:04:52 2008 -0500

    Stop gssd from ignoring the machine credential cache
    defined by the -d flag
    
    Signed-off-by: Steve Dickson <steved@redhat.com>
    Signed-off-by: Kevin Coffman <kwc@citi.umich.edu>

diff --git a/utils/gssd/krb5_util.c b/utils/gssd/krb5_util.c
index bf8690e..3cf27ca 100644
--- a/utils/gssd/krb5_util.c
+++ b/utils/gssd/krb5_util.c
@@ -404,7 +404,7 @@ gssd_get_single_krb5_cred(krb5_context context,
 	    cache_type = "FILE";
 	snprintf(cc_name, sizeof(cc_name), "%s:%s/%s%s_%s",
 		cache_type,
-		GSSD_DEFAULT_CRED_DIR, GSSD_DEFAULT_CRED_PREFIX,
+		ccachedir, GSSD_DEFAULT_CRED_PREFIX,
 		GSSD_DEFAULT_MACHINE_CRED_SUFFIX, ple->realm);
 	ple->endtime = my_creds.times.endtime;
 	if (ple->ccname != NULL)
